src/components/network/NetworkItem.vue

```diff
@@ -4,7 +4,7 @@
     :class="{ active: connection.network === network.id }"
   >
     <div class="network-icon">
-      <img :src="network.logo_url" alt="cosmic atom token" />
+      <img src="https://s3.amazonaws.com/network.logos/cosmos-logo.png" alt="cosmic atom token" />
```

Comment:
this is wrong. the logos are stored locally in Lunie not in a remote repo

Reply:
Of course, will update.

Reply:
👀